src/main/java/com/adamk33n3r/runelite/watchdog/ui/notifications/panels/NotificationPanel.java

```diff
@@ -11,16 +13,15 @@
 import net.runelite.client.ui.components.MouseDragEventForwarder;
 
 import lombok.Getter;
+import net.runelite.client.util.ImageUtil;
 
-import javax.swing.BorderFactory;
-import javax.swing.JButton;
-import javax.swing.JLabel;
-import javax.swing.JPanel;
+import javax.swing.*;
 import javax.swing.border.Border;
 import javax.swing.border.CompoundBorder;
 import javax.swing.border.EmptyBorder;
-import java.awt.BorderLayout;
-import java.awt.FlowLayout;
+import java.awt.*;
```

Comment:
For future reference, I have my settings to only wildcard imports if it's using 5 or more. I also have some other rules about where the imports go. I wonder if those can be committed to the repo...